bin/common.sh

```diff
@@ -65,9 +84,9 @@ function _rind() {
   fi
 }
 
-#######################################
-# https://github.com/google/styleguide/blob/gh-pages/shellguide.md#quoting
-#######################################
+#####
```

Comment:
I propose we should agree on the size of the function comments and add it to the style guide. 5 seems enough so one doesn't have to think about it a lot and it's still visible when skimming a script.

Reply:
3 is also fine with me then...but I think ppl will just copy paste